Bundle/BlogBundle/Resources/views/Blog/Tabs/_drafts.html.twig

```diff
@@ -0,0 +1,63 @@
+{% trans_default_domain "victoire" %}
```

Comment:
don't you think you should extend the `Blog:Tabs/_articles.html.twig` template ?